packages/wdio-spec-reporter/tests/index.test.ts

```diff
@@ -395,7 +396,7 @@ describe('SpecReporter', () => {
             expect(result).toMatchSnapshot()
         })
 
-        it('should not print if data table format is not given', () => {
+        it('should not print if argument is a single line doc string', () => {
             tmpReporter.getOrderedSuites = vi.fn(() => {
                 const suites = Object.values(JSON.parse(JSON.stringify(SUITES_WITH_DATA_TABLE))) as any[]
                 suites[0].hooksAndTests[0].argument = 'some different format'
```

Comment:
The current code support only data table (type of Argument) and string can be used to test for wrong data table format input. The changing for doc string supports will not allow this as the argument is a string for doc string.